sre/kvStorage.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import "fmt"
+
+func check(e error) {
```

Comment:
Can get rid of this function. We don't want to panic from inside a library. Panic is always done at the top most level. A storage library is the backend.

Ex: A CLI service -> has an commad `myservice save --value` -> which calls your storage library's function called save()

now this save() func should return an error. which is captured by the command `save` and this command can decide to either panic, or return the error in stdout/stderr.